apps/portal/js/designer.js

```diff
@@ -1172,7 +1172,7 @@ $(function () {
         }
 
         if(sandbox.context.className == "form-control title"){
-            if(checkForPagesByTitle(title) && page.title != title){
+            if(checkForPagesByTitle(title) && page.title.toUpperCase() != title.toUpperCase()){
                 generateMessage("error","A page with entered title already exists. Please select a different title");
```

Comment:
Formatting issue. There should be a space for ','. And use locales without hard coding the messages.